pkg/version/version.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2020 Kohl's Department Stores, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+import (
+	"fmt"
+	"runtime"
+)
+
+// Application build information.
+var (
+	Branch    string
+	BuildDate string
+	GitSHA1   string
+	Version   = "v0.0.1-dev"
+)
+
+// Print writes application version details to standard output.
+func Print() {
+	// TODO remove hard coded "prometheus_bigquery_remote_storage_adapter" string here
+	fmt.Println(Get())
+}
+
+// Get returns the version string with some additional details
+func Get() string {
```

Comment:
Two things ...

Do you think the `Get` function is needed? I think this functionality could moved into the `Print` function.

If you decide to keep the `Get` function I'd recommend renaming it to `get`, so that it is not exported.

Reply:
I created the get function so that I can use this in log messages and on the console itself. I guess yeah I could completely remove the console message, since it'll come through the log message anyway.

Reply:
@seanmalloy question: You're suggesting making it lowercase, but the `Print` function I copied from git2consul uses an uppercase P. Which standard should we settle on?

Reply:
Variables, functions, and methods that start with an upper case letter are exported. See https://tour.golang.org/basics/3.

If you want to keep the `Get` function and need to call it from outside of the `version` package then it needs a capital `G`. Otherwise stick with a lowercase `g` if it will only be used from within the `version` package.

Reply:
Ah! Thanks for the info. Yeah I need it exported, so I'll leave it in there.